model/resource/process.yaml

```diff
@@ -1,6 +1,7 @@
 groups:
   - id: process
     type: resource
+    name: process
```

Comment:
it's unfortunate that yaml type is `resource` and proto resource type is `name`.

How much would you hate `resource_type` instead of the `name` ?

I know we want a common identity property on all groups, but `name` is problematic on spans anyway

Reply:
There's a possibility we rename this to `entity` so I'd like to avoid major refactorings right now.